lib/rubocop/rspec/language.rb

```diff
@@ -159,9 +159,14 @@ def context(element)
       end
 
       module Runners # :nodoc:
-        ALL = %i[to to_not not_to].freeze
-        def self.all(element)
-          ALL.include?(element)
+        class << self
+          def to_a
+            %i[to to_not not_to].freeze
+          end
+
+          def all(element)
+            to_a.include?(element)
+          end
```

Comment:
This will create a new array each time. Why not just defining `to_a = ALL`

Reply:
Indeed it is. I updated it. Thank you so much.